<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLTMultimeterForm</class>
 <widget class="QWidget" name="DLTMultimeterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DLT Multimeter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Power</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditPowerName"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxOn">
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Value</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEditValueV1"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEditValueA1"/>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBox">
        <property name="specialValueText">
         <string>12.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditUnitV1"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditUnitA1"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonSet">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
